src/include/cell.i

```diff
@@ -933,6 +1040,7 @@ __wt_cell_unpack_dsk(
      * Previous startup txnid=0, ts=y       txnid=0, ts=WT_TS_NONE           txnid=MAX, ts=MAX
      */
     if (dsk->write_gen > 0 && dsk->write_gen <= S2C(session)->base_write_gen) {
```

Comment:
@bvpvamsikrishna, do you have an idea about this code, whether we need to clear durable timestamps from a previous run?

Reply:
@ddanderson, we should not clear the durable timestamps from a previous run. Timestamps are not reused across restarts. We only clear the txn id's as we are reusing them across restarts.

Reply:
@bvpvamsikrishna I still think something needs to happen down here. If you have a transaction id but no timestamp, we set the timestamp to `WT_TS_NONE` rather than `WT_TS_MAX`. Wouldn't you need to do the same thing with the durable timestamp?

Reply:
@tetsuo-cpp, that makes some sense to me.  I've made the change so we have something to discuss.  There's an assert in that code that may also apply to the durable timestamps.